src/sentry/seer/code_review/webhooks/handlers.py

```diff
@@ -13,26 +13,20 @@
 if TYPE_CHECKING:
     from sentry.integrations.github.webhook import WebhookProcessor
 
+from ..permissions import has_code_review_enabled
 from ..utils import _transform_webhook_to_codegen_request
 from .check_run import handle_check_run_event
+from .issue_comment import handle_issue_comment_event
 
 logger = logging.getLogger(__name__)
 
 METRICS_PREFIX = "seer.code_review.webhook"
 
 
-def handle_other_webhook_event(
-    *,
-    event_type: str,
-    event: Mapping[str, Any],
-    organization: Organization,
-    repo: Repository,
-    **kwargs: Any,
+def _forward_to_seer(
```

Comment:
Let's call it this for now:
```suggestion
def _schedule_task(
```